jwst/master_background/master_background_step.py

```diff
@@ -17,9 +17,9 @@ class MasterBackgroundStep(Step):
     spec = """
         user_background = string(default=None) # Path to user-supplied master background
         save_background = boolean(default=False) # Save computed master background
+        subtract_background = boolean(default=None) #control flag
```

Comment:
Ideally the comment here should provide a user with a hint as to what the param does, so I would change the wording from "control flag" to something like "Subtract master background".